.bit/responses/4.6-Week 4 Step 7.md

```diff
@@ -0,0 +1,20 @@
+---
+files: n/a
+stepType: checks
+scripts: 
+week: 4
+step: 7
+name: Week 4 Step 7
+---
+
+Week 4 Step 7 ⬤⬤⬤⬤⬤⬤⬤ | 🕐 Estimated completion: 5-20 minutes
+
+## Ultimate CAT INVASION! ~
+
+Demo: [EMILY GO INSERT A DEMO HERE]
+
+### ✅  Task:
+- [ ] Connect everything together!(twocatz/index.html, twocatz/site.js)
```

Comment:
Be a little more specific lol like
*also describe the end goals of the project, or else they have no idea what they're doing*
- Make sure you're on the week 4 branch
- Commit to ____
- Create 4 input boxes to accept 4 names
- Retrieve values from the 4 input boxes
- Create a GET request with 4 names

For this one you also need to explain how to display base64 on an HTML page